docs/learn/uses-of-tezos/nft.md

```diff
@@ -24,23 +24,22 @@ Recently, NFTs have been a powerful primitive demonstrating usefulness in the cr
 * [tzNFT](https://github.com/tqtezos/nft-tutorial) - a tutorial showing users how to originate and interact with the FA2 NFT contract implementation.
 * [TZIP-021](https://gitlab.com/tzip/tzip/-/blob/tzip-21-spec/proposals/tzip-21/tzip-21.md) - an emerging contract multimedia metadata standard. 
 
-### Tools and Applications
-
-* \*\*\*\*[Truesy](https://www.truesy.com/) ****- a highly curated NFT marketplace
-* [Kalamint](https://kalamint.io/) - A community owned NFT marketplace on Tezos.
-* [OpenMinter](https://github.com/tqtezos/minter) ****and [Minter-SDK](https://github.com/tqtezos/minter-sdk) - A reusable dApp and SDK that allows anyone to create, collect, sell, or swap NFTs on Tezos.
-* \*\*\*\*[tzcolors](https://www.tzcolors.io)\*\*\*\*
-* \*\*\*\*[TezAuction](https://tez.auction/#/)\*\*\*\*
-* [CricTez](https://crictez.medium.com/introducing-crictez-21a284c60e94)
+### NFT Marketplaces 
```

Comment:
Please remove descriptions for Objkt, Kalamint, and tzcolors

Reply:
Sorry I'm learning Github, idk how to edit the code.